events/api.py

```diff
@@ -777,7 +843,9 @@ def validate(self, data):
 
         # require the publication status
         if 'publication_status' not in data:
-            raise serializers.ValidationError({'publication_status': 'You must specify whether you wish to submit a draft or a public event.'})
+            raise serializers.ValidationError({'publication_status':
+                                                   _(
+                                                       "You must specify whether you wish to submit a draft or a public event.")})
```

Comment:
Oh, it does. PyCharm apparently does not know how to implement PEP 8 recommendations here.